### DISCUSSION TITLE
Details about the .csv files to generate rotors

### DISCUSSION THREAD

**Discussion by NM1293** [byuflowlab/FLOWUnsteady#149] [Q&A] [answered]
Hello everyone,

I have a simple question regarding some of the .csv files used to implement the rotor in the code, specifically sweepdist.csv, pitchdist.csv, heightdist.csv, and chorddist.csv. I think I know the answer, but it's better to remove any doubt and ensure that the rotor is generated correctly.

The question is: Can the number of rows in each of these files be any? For example, the chorddist.csv file consists of 26 rows (i.e., 26 values of c/R, each associated with an r/R). If I want to generate a new personal rotor for which .csv files are not included in your database, and thus I have to create them myself, is the number of rows in the sweepdist.csv, pitchdist.csv, heightdist.csv, and chorddist.csv files irrelevant?

Thank you for your help, and for this useful tool.

**Comment by cibinjoseph** [comment, marked as answer]
@NM1293 Yes, the number of rows can be different in those files when specifying the blade geometry.
FLOWUnsteady performs a spline fit to each of those distributions and uses that curve to create the same number of blade sections for each of those parameters.
